packages/amplify-util-mock/src/__tests__/api/api.test.ts

```diff
@@ -82,12 +84,15 @@ describe('Test Mock API methods', () => {
     } as unknown as $TSContext;
 
     const testApi = new APITest();
-    const testApiStartPromise = testApi.start(mockContext);
+    await testApi.start(mockContext);
 
-    await expect(testApiStartPromise).rejects.toThrow(
+    await expect(testApi['getAppSyncApiPublic'](mockContext)).rejects.toThrow(
       new AmplifyFault('MockProcessFault', {
-        message: 'Failed to start API Mocking.. Reason: No AppSync API is added to the project',
-      }),
+        message: 'No AppSync API is added to the project',
+        resolution: `Use 'amplify init' in the root of your app directory to create a new environment.`,
+        link: 'https://docs.amplify.aws/cli/graphql/troubleshooting/',
+      })
```

Comment:
true, wanted to test `getAppSyncApi` method when it fails for not able to fetch the Api. SO, extracted the method so it could be used in testing to assert the exact behavior